discovery-infra/install_cluster.py

```diff
@@ -94,6 +94,7 @@ def run_install_flow(
         pull_secret,
         tf,
         cluster_deployment=None,
+        agent_cluster_install=None,
         nodes_number=None,
 ):
     if cluster_deployment is None:
```

Comment:
but now we should verify agent_cluster_install? no?

Reply:
When cluster_deployment support was added we broke make install_cluster.Maybe it should be fixed?